web/index.php

```diff
@@ -2,12 +2,16 @@
 
 declare( strict_types = 1 );
 
+use Symfony\Component\Dotenv\Dotenv;
 use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
 use WMDE\Fundraising\Frontend\App\Kernel;
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
+// We set the env from the .env file instead of web server settings because we want to be independent from the hosting provider
+( new Dotenv() )->bootEnv( __DIR__ . '/../.env' );
```

Comment:
Thought you didn't want to use .env files?

Reply:
env files are not the best way to keep secrets, but with our environment (web server config controlled by hoster, no containers where you could pass them in) and the requirement that we want to be able to change them without involving the hoster, I think they're the best solution we have at the moment. If you have suggestions on different approaches, I'd like to hear them, but for the time being (i.e. this PR) we go with env files.

I'll have another look at the deploy playbook to set the permissions as minimal as possible (only readable by deploy uder and www-data group).